topics/contributing/tutorials/fair-by-design/tutorial.md

```diff
@@ -0,0 +1,1065 @@
+---
+layout: tutorial_hands_on
+title: FAIR-by-Design methodology
+abbreviations:
+  FAIR: Findable, Accessible, Interoperable, Reusable
+  OER: Open Educational Repositories
+  EOSC: European Open Science Cloud
+  CC: Creative Commons
+  MVS: Minimum Viable Skills
+  TeSS: Training eSupport System
+zenodo_link: https://zenodo.org/records/11548062
+questions:
+  - How to develop FAIR learning materials?
+  - How to incorporate the FAIR principles in the instructional design process?
+objectives:
+  - Design FAIR learning materials
+  - Structure FAIR learning materials
+  - Create and publish FAIR-by-Design learning materials
+  - Assess FAIR-ness of learning objects
+time_estimation: 20M
+key_points:
+  - The FAIR-by-Design Methodology ensures that the output of the instructional design process are FAIR learning materials.
+  - Each stage of the FAIR-by-Design Methodology incorporates specific parts of the FAIR principles.
+  - The ouput of the FAIR-by-Design Methodology are FAIR learning materials from both the perspective of instructors and of learners.
+tags:
+  - 'FAIR-by-Design Learning Materials'
+  - "FAIR Learning Objects"
+  - FAIR-by-Design Methodology
+priority: 1
+contributions:
+  authorship:
+    - sonjafiliposka
+    - amisev1971
+    - CarolinLeister
+    - korvoj
+    - domlgreen
+    - isouyioul
+    - achim-kit
+    - agCleo
+    - CRHAD26
+  funding:
+    - skills4eosc
+follow_up_training:
+ - type: external
+   title: Skill4EOSC FAIR-by-design Methodology Training of Trainers
+   link: "https://learning.skills4eosc.eu/course/view.php?id=19"
+lang: "en"
+level: "Introductory"
+extra:
+  description: "Short how to guide that guides you through the stages of the FAIR-by-Design methodology without any specific choice on tools and formats."
+  target_audience: "Instructional designers"
+  URL_resource_type: "Permanent URL"
+  learning_resource_type: "self-paced microlearning unit"
+  access_cost: "No"
+  attribution: "This microlearning unit is based on Filiposka, Sonja, Mishev, Anastas. (2024). FAIR-by-Design Microlearning (1.0.0). https://doi.org/10.5281/zenodo.11548062"
```

Comment:
hmm. 

`description` should be able to be set at the top level but it isn't used for anything except the social media preview.

`target_audience` is a key we should consider supporting.

What is the purpose of `url_resource_type`? `access_cost` this metadata isn't really exposed anywhere, if that is important enough to include here, it's something to discuss.

`attribution` we traditionally do in a large box after the introduction (e.g. https://training.galaxyproject.org/training-material/topics/contributing/tutorials/learning-principles/tutorial.html#comment-sources ) since again, nothing in `extra` is exported anywhere.

Reply:
According to the Skills4EOSC Fair-by-design Methodology, which uses the RDA Minimal Metadata Schema, all fields should be provided in both human and machine readable format. The schema defines: url_resource_type, access_cost, therefore we need to include them.

Reply:
> all fields should be provided in both human and machine readable format. 

exactly! and all of the fields you provided are not exposed anywhere human readable, nor machine readable. extras is only visible in this markdown file, nowhere else, nowhere in the rendered tutorial. 

So it has to be a discussion between us of how and where we expose that information. 

some keys are specific to your tutorial (the source), and some are not: For instance all tutorials on the GTN are freely available, so maybe we find somewhere to expose that information for every tutorial and you don't need to set it here.

Reply:
@amisev1971 morning, looking into this further today :) I see [this handbook page](https://fair-by-design-methodology.github.io/microlearning/latest/01%20Prepare/prepare/#__tabbed_1_2) which references https://zenodo.org/records/6769695#.YrrP9-xBybQ but looking through that I'm not seeing an specification they expect, nor any specific format. E.g. Schema.org or Dublin Core or any similar controlled vocabulary.

Could you help us figure that out? 

The GTN currently exposes a *wealth* of metadata via ld+json (our preference) and a subset of that is available via dublin core terms in our page metadata (also visible in the agenda box.)

Specifically our learning materials adhere to the https://schema.org/LearningResource profile.

Your request | Schema.org/LearningResource supports | Actions
--- | --- | ---
description | yes!  | (You) Please move this to the top level `description` (i.e. just outside of `extras`)
target audience | yes! | (GTN) needs to add support for customising this, then we'll add to this PR to move it.
URL_resource_type: "Permanent URL" | no, schema.org does not support | based on [the prepare section](https://fair-by-design-methodology.github.io/microlearning/latest/01%20Prepare/prepare/#__tabbed_1_2)  I'm not sure this needs to be explicitly annotated: we don't use DOI/ARK or handles, instead just permanent URLs. Since the URL to the page is annotated in the machine readable metadata, perhaps we could consider omitting this? The URL is clear that it's a URL via `https://`, rather than `doi://` or another protocol?
learning_resource_type: "self-paced microlearning unit" | yes, [`learningResourceType`](https://schema.org/learningResourceType). | We already set this, for this it will be "hands-on tutorial".  (GTN) could add support for customising.
access_cost: "No" | no, not supported by schema.org. | I'm not sure how best to expose this, since *100%* of our materials have no access cost I would rather set something globally for the entire GTN, rather than just for this specific material, or needing to set it on every material. Open to suggestions for how best to expose this. Other schema.org schemas like Course have an associated cost field, but not a LearningResource.
attribution: ... | no, not supported by schema.org | I'm not sure how we could make this machine readable, the text you provided for instance doesn't break that information down into author/url/license/etc. So if you actually want it to be machine readable it'd need to be broken down further.

Reply:
Hi @hexylena, the RDA Minimal Metadata Schema does not provide formal specification and we understand that this can be a problem (it was also a problem for us). In our gitpages we provide all the metadata fields defined in the schema in the md header and md body of the syllabus. The header is then rendered into meta fields in the html head section. Using all the fields from the RDM Minimal Metadata Schema is a strong requirement in the project consortium of Skills4EOSC, and we did our best to implement it somehow in your platform, using the extra field. If you are considering making this visible, maybe adding a collapsing box for the extra fields in the overview tab would be a possible solution.

Reply:
Hi @amisev1971 do you have a link to one of those pages? I'd love to look at it!

Reply:
The landing pages of the micro-learning (https://fair-by-design-methodology.github.io/microlearning/latest/) and the Train of Trainers (https://fair-by-design-methodology.github.io/FAIR-by-Design_ToT/latest/) are two such examples. In addition, they also use FAIR Signposting. The micro-learning pages use Level 1 signposting, while the Train of Trainers uses Level 2 signposting. To view the meta information, right click on the landing page and select the Show page source (Safari) or View page source (Chrome).